packages/rps/src/redux/message-service/saga.ts

```diff
@@ -49,20 +49,18 @@ type Message = AppMessage | WalletMessage;
 
 export function* sendWalletMessageSaga() {
   const sendMessageChannel = createWalletEventChannel([Wallet.MESSAGE_RELAY_REQUESTED]);
-  while (true) {
-    const messageRelayRequest: MessageRelayRequested = yield take(sendMessageChannel);
-
-    const { messagePayload, to } = messageRelayRequest;
-    const messageToSend: WalletMessage = { payload: messagePayload, queue: Queue.WALLET };
-
-    yield fork(
-      reduxSagaFirebase.database.create,
-      `/messages/${to.toLowerCase()}`,
-      sanitizeMessageForFirebase(messageToSend),
-    );
-  }
+  yield takeEvery(sendMessageChannel, sendMessageSaga);
```

Comment:
This change was required to handle the case where two `commitments_received` were queued up in the outbox.